src/forms/renderers/ConnectorAutoComplete.tsx

```diff
@@ -0,0 +1,138 @@
+/*
+  The MIT License
+
+  Copyright (c) 2017-2020 EclipseSource Munich
+  https://github.com/eclipsesource/jsonforms
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  THE SOFTWARE.
+*/
+import { EnumCellProps, EnumOption, WithClassname } from '@jsonforms/core';
+import {
+    Autocomplete,
+    AutocompleteRenderOptionState,
+    Box,
+    FilterOptionsState,
+    Input,
+} from '@mui/material';
+import ConnectorName from 'components/ConnectorName';
+import merge from 'lodash/merge';
+import React, { ReactNode } from 'react';
+
+export interface WithOptionLabel {
+    getOptionLabel?(option: EnumOption): string;
+    renderOption?(
+        props: React.HTMLAttributes<HTMLLIElement>,
+        option: EnumOption,
+        state: AutocompleteRenderOptionState
+    ): ReactNode;
+    filterOptions?(
+        options: EnumOption[],
+        state: FilterOptionsState<EnumOption>
+    ): EnumOption[];
+}
+
+export const ConnectorAutoComplete = (
```

Comment:
Brought in the MuiAutoComplete from JsonForms so we could tweak. They do not expose it - hence copying it into our codebase.